### ISSUE TITLE
s.remove_elements() does not exist for EELS or EDS

### ISSUE THREAD

**Issue by thomasaarholt** [hyperspy/hyperspy#1337]
EELS and EDS signals do not have a method to remove added elements (which are added by `s.add_elements()`), which makes it inconvenient if one has accidentally added the wrong element to a signal.
`s.metadata.Sample.elements = []` does not work, nor does `s.elements.clear()`.

**Comment by francisco-dlp** [comment]
Implementing this should be as simple as sharing the current functionality implemented for EDS with the EELS class.

**Comment by thomasaarholt** [comment]
I couldnt find it for eds. are you sure it exists?

**Comment by francisco-dlp** [comment]
You are right, they don't exist in EDS either, what exists is [set_elements](https://github.com/hyperspy/hyperspy/blob/RELEASE_next_minor/hyperspy/_signals/eds.py#L235) which is a combination of erasing all elements and adding new ones. However, in EDS, unlike in EELS, you could simply do `s.metadata.Sample.elements.remove("Fe")` to remove an element, while for EELS this doesn't work. In any case, my comment was misleading, it is actually not that straightforward to implement this feature, but it shouldn't be very hard either.
